src/main/java/marquez/dao/JobRunStateRow.java

```diff
@@ -3,17 +3,17 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.UUID;
+import marquez.core.models.JobRunState;
```

Comment:
this is a good change, but seems totally unrelated to this PR

Reply:
Yup, you're right. Will keep PRs discrete.